src/features/metadata-based-view/MetadataBasedItemList.js

```diff
@@ -168,7 +196,7 @@ class MetadataBasedItemList extends React.Component<Props, State> {
                 className={classes}
```

Comment:
Unrelated to the changeset, but do we want to set the `'bdl-MetadataBasedItemList-cell--hover'` class even if it's a header row?

Reply:
no. So far, this class is only used for adding background color, and we don't need bgColor for header row.